services/tasks/local.go

```diff
@@ -255,6 +261,16 @@ func (l *local) Create(ctx context.Context, r *api.CreateTaskRequest, _ ...grpc.
 	}, nil
 }
 
+func (l *local) emitRuntimeWarning(ctx context.Context, runtime string) {
+	switch runtime {
+	case plugin.RuntimeLinuxV1:
```

Comment:
so we are only emitting warnings for io.containerd.runtime.v1.linux. Are we not deprecating other v1 runtimes like io.containerd.runtime.v1.windows?

Reply:
That's a good question.  Does `io.containerd.runtime.v1.windows` exist?  I see it in a couple places in the client code ([1](https://github.com/containerd/containerd/blob/76395d2a26f2fdf4173c2ef498afe7f332a77083/client.go#L188), [2](https://github.com/containerd/containerd/blob/76395d2a26f2fdf4173c2ef498afe7f332a77083/task.go#L315), [3](https://github.com/containerd/containerd/blob/76395d2a26f2fdf4173c2ef498afe7f332a77083/task.go#L332)) but can't find the actual plugin implementing it.  @dcantah do you know?